common/lib/xmodule/xmodule/js/spec/video/html5_video_spec.js

```diff
@@ -105,8 +103,9 @@
                         jasmine.waitUntil(function() {
                             return state.videoPlayer.player.getPlayerState() !== STATUS.PAUSED;
                         }).then(function() {
-                            expect(state.videoPlayer.player.getPlayerState())
-                                .toBe(STATUS.BUFFERING);
+                            expect([STATUS.BUFFERING, STATUS.PLAYING]).toContain(
```

Comment:
This change is required because there is no guarantee that video state will be `STATUS.BUFFERING` after video starts playing, the state could be `STATUS.BUFFERING` or `STATUS. PLAYING`. Thats why sometime this test fails and sometime it passes.